docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '2'
+services:
+  elasticsearch:
+    image: elasticsearch:latest
+    ports:
+      - "${ELASTICSEARCH_PORT}:${ELASTICSEARCH_PORT}"
+  mongodb:
+    image: mongo:latest
```

Comment:
You should lock to a specific version. Production is running 3.0.6, so I would set it to

`mongo:3.0`